### ISSUE TITLE
Restore /concrete for malicious attacks

### ISSUE THREAD

**Issue by dbuonomo** [mnakalay/Concrete5-Super-User-recovery#2]
If there was a malicious attack, something in the Concrete core may have been changed. So, it's best to delete and restore /concrete. This could be by pulling it from your repository (if you have one), or downloading it from the Concrete CMS website.

**Comment by mnakalay** [comment]
Yes, I agree. It is, however, a bit out of scope for this tool. It deals with user account tempering, mostly in the database.
